misc/services/camel/de-metas-camel-externalsystems/core/readme.md

```diff
@@ -0,0 +1,24 @@
+
+### Metasfresh => Camel External System Dispatcher
+
+ - the entry point for any message is the `rabbitMQ` camel route consumer defined as: `rabbitmq:MF_TO_ExternalSystem`, with the following options:
+   - `durable=true`
+   - `autoDelete=false`
+   - `prefetchCount=1`
+     - *The maximum number of messages that the server will deliver* 
+     - `prefetchEnabled=true` (mandatory complementary option to `prefetchCount`):
+       - *Enables the quality of service (QOS) on the RabbitMQConsumer side* 
+     - `prefetchGlobal=false` (mandatory complementary option to `prefetchCount`):
+       - *If the settings should be applied to the entire channel rather than each consumer* 
+     - `prefetchSize=0` (mandatory complementary option to `prefetchCount`): 
+       - *The maximum amount of content (measured in octets) that the server will deliver, 0 if unlimited* 
```

Comment:
this is also unclear to me. in both cases, please either give a more understandable documentation or add a reference to the documentation at rabbitmq or camel